<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddURL</class>
 <widget class="QDialog" name="AddURL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <widget class="QTabWidget" name="inputTabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="enter_url_tab">
        <attribute name="title">
         <string>Enter URL</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QFormLayout" name="url_form">
           <item row="0" column="0">
            <widget class="QLabel" name="url_dest_label">
             <property name="text">
              <string>Destination: </string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="url_label">
             <property name="text">
              <string>URL: </string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="url_hash_label">
             <property name="text">
              <string>Hash: </string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="url_descLabel">
             <property name="text">
              <string>( http(s)://, ftp(s)://, magnet: )</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="url_dest_horizLayout">
             <item>
              <widget class="QLineEdit" name="url_dest_lineEdit">
               <property name="maxLength">
                <number>4096</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="url_dest_toolButton">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="1">
            <widget class="QPlainTextEdit" name="url_plainTextEdit">
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOn</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="plainText">
              <string notr="true"/>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::LinksAccessibleByMouse|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="url_hash_lineEdit">
             <property name="maxLength">
              <number>1024</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="open_file_tab">
        <attribute name="title">
         <string>Open a File</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QFormLayout" name="file_form">
           <item row="0" column="0">
            <widget class="QLabel" name="file_dest_label">
             <property name="text">
              <string>Destination: </string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="file_import_label">
             <property name="text">
              <string>File to import: </string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="file_dest_horizLayout">
             <item>
              <widget class="QLineEdit" name="file_dest_lineEdit"/>
             </item>
             <item>
              <widget class="QToolButton" name="file_dest_toolButton">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="file_import_horizLayout">
             <item>
              <widget class="QLineEdit" name="file_import_lineEdit"/>
             </item>
             <item>
              <widget class="QToolButton" name="file_import_toolButton">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="file_import_desc_label">
             <property name="text">
              <string>( e.g. download.torrent, download.nzb, urls.txt )</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddURL</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddURL</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
